linker.go

```diff
@@ -58,7 +58,7 @@ func TestLinker(t *testing.T, store types.Storager) {
 				linkTarget, ok := o.GetLinkTarget()
 
 				So(ok, ShouldBeTrue)
-				So(linkTarget, ShouldEqual, target)
+				So(linkTarget, ShouldEndWith, target)
```

Comment:
How about always use the absolute path for `LinkTarget`?

Reply:
We can set the absolute path while setting link target to the object. But `target` here is not a absolute path, so `linkTarget` and `target` may not equal.

Reply:
> We can set the absolute path while setting link target to the object. But `target` here is not a absolute path, so `linkTarget` and `target` may not equal.

oops, I got you.

How about `So(linkTarget, ShouldEqual, workdir + target)`?